dashboard/src/components/model/Inverter.vue

```diff
@@ -24,30 +39,39 @@
 </template>
 
 <script lang="ts">
+import ModelBase from "@/components/ModelBase.vue";
+import HelpPopup from "@/components/Help.vue";
+import InverterParametersView from "@/components/model/InverterParameters.vue";
+import LossParametersView from "@/components/model/LossParameters.vue";
+import ArraysView from "@/components/model/Arrays.vue";
+
 import { Component, Prop, Vue, Watch } from "vue-property-decorator";
-import InverterParametersView from "@/components/InverterParameters.vue";
-import ArraysView from "@/components/Arrays.vue";
 import { Inverter } from "@/types/Inverter";
+import { PVWattsLosses } from "@/types/Losses";
 import {
-  PVSystInverterParameters,
+  SandiaInverterParameters,
   PVWattsInverterParameters
 } from "@/types/InverterParameters";
 
 Vue.component("arrays-view", ArraysView);
 Vue.component("inverter-parameters", InverterParametersView);
+Vue.component("loss-parameters", LossParametersView);
+Vue.component("help", HelpPopup);
```

Comment:
Should all of these components be registered globally or at a higher level instead of in each component?

https://vuejs.org/v2/guide/components-registration.html#Global-Registration